adaezeodurukwe/iReport
# #162470369 Add more tests

**What does this PR do?**
Adds more tests

**Description of work done?**
- Test for errors on records route
- Test user routes

**How can this be tested?**
After cloning the repo, cd into the folder,  checkout to the ch-write-more-tests-162470369  branch
- run npm test

All tests should pass

**What are the relevant PT stories**
#162470369
